boards/arm/rpi_pico/rpi_pico.dts

```diff
@@ -84,3 +86,17 @@
 	pinctrl-0 = <&i2c1_default>;
 	pinctrl-names = "default";
 };
+
+&spi0 {
+	clock-frequency = <DT_FREQ_M(8)>;
+	status = "okay";
+	pinctrl-0 = <&spi0_default>;
+	pinctrl-names = "default";
+};
+
+&spi1 {
```

Comment:
Per my other comment, I think we shouldn't be setting up `spi1` at all with defaults, leave that to end users.

Reply:
ditto.